cluster/sync.sh

```diff
@@ -1,15 +1,15 @@
 #!/bin/bash -ex
 
 source ./hack/common.sh
+source ${PWD}/_kubevirtci/cluster-up/cluster/ephemeral-provider-common.sh
```

Comment:
Use same path style as above? `.` instead of `${PWD}`?

Reply:
I agree.